### DISCUSSION TITLE
I'm Not That Dumb, But I Feel Like It (Commands)

### DISCUSSION THREAD

**Discussion by Kuroo2021** [Tyrrrz/DiscordChatExporter#1316] [Q&A]
![image](https://github.com/user-attachments/assets/a1b8f0fc-9158-46bb-9f01-b984aa9c94d4)

Okay I get how to do the basic but it seems I not doing this correctly.  

Even following the guides at https://github.com/Tyrrrz/DiscordChatExporter/blob/master/.docs/Using-the-CLI.md

Could someone maybe explain what I may be doing wrong?

**Comment by 96-LB** [comment]
If you're not familiar with using command-line apps, I recommend you download and use the GUI version instead. It'll be a much mor straightforward experience.

If you must use the CLI for some reason, could you share the command you ran before the screenshot to get the first output? My guess is that your `DiscordChatExporter.Cli` file is in a different folder than the one in which you're running these commands. When using Windows Powershell, you should first run a command like `cd C:\path\to\dce\folder` before trying to run the program, and then you can easily run it using `.\DiscordChatExporter.Cli`.

**Reply by Kuroo2021** [reply]
It running from the same window as I open up the program first with in the window I just didn't add it to the screenshot so I don't have to edit the image for private stuff. I'm running the commands and such after loading the . I loaded the DiscordChatExporter.Cli.exe file first. then try the command afterwards.

**Reply by 96-LB** [reply]
You have to "load" the file and run the command on the same line of input. You should be adding `--version` or `export ...` or whichever command you want to the end of the line in which you run the exe.

**Reply by Kuroo2021** [reply]
Does it not show then I have load the software when given the prompts telling me my options? When running the command I can't seem to get it to work as shown above.

**Reply by 96-LB** [reply]
Every command you run loads the software, runs the command, and then closes the software. The prompt with options you are seeing is the output you get if you don't specify a command or use `-h`.

Please share the commands you are running (with your token/other private info censored).

**Comment by Tyrrrz** [comment]
Try `./DiscordChatExporter.Cli ...`

**Reply by Kuroo2021** [reply]
![image](https://github.com/user-attachments/assets/85c97d50-5dea-4921-bfc1-0ce4726b0ad9)

Still nothing, just an error.

**Reply by 96-LB** [reply]
When you run `.\DiscordChatExporter.Cli`, PowerShell is looking for the file in the current folder, which is `C:\Users\username`. This is not where you downloaded your file. You should do one of two things:
1. Give the full path to the file in every command like you did in the first one. This means your commands would look like `C:\Users\username\DiscordChatExporter.Cli.win-x64\DiscordChatExporter.Cli.exe -h` and not the shorter version you're trying.
2. Before you run `.\DiscordChatExporter.Cli`, you have to run a `cd` command to move folders. Run `cd C:\Users\username\DiscordChatExporter.Cli.win-x64` before you do any DCE commands.

Of course, replace `username` with your actual username in the above commands.
